src/components/Card.js

```diff
@@ -15,6 +15,18 @@ const Wrapper = styled.div`
   a {
     text-decoration: none;
   }
+
+  img {
+    max-width: 100%;
+  }
+
+  @media screen and (min-width: 500px) {
+    flex: 0 0 50%;
+  }
+
+  @media screen and (min-width: 770px) {
+    flex: 0 0 33.333%;
+  }
```

Comment:
These styles relate to Wrapper so it's better to move them up to Wrapper styles (above other selectors)

Reply:
Done